oak-search/src/test/java/org/apache/jackrabbit/oak/plugins/index/IndexQueryCommonTest.java

```diff
@@ -279,7 +279,7 @@ public void containsPathNum() throws Exception {
         a.setProperty("name", "/segment1/segment2/segment3");
         root.commit();
 
-        String stmt = "//*[jcr:contains(., '/segment1/segment2')]";
+        String stmt = "//*[jcr:contains(., '\\\\/segment1\\\\/segment2')]";
```

Comment:
Would the user need to escape characters like this while querying on paths ?

Reply:
I suppose Fabrizio's comment above regarding FulltextIndex.rewriteQueryText(text) would handle this and user might not need to escape in queries - right ?

Reply:
@nit0906 agreed, this should be reverted after using FulltextIndex.rewriteQueryText(text), I will fix it.